### DISCUSSION TITLE
List ArchR on Bioconductor

### DISCUSSION THREAD

**Discussion by Pandapip1** [GreenleafLab/ArchR#1892] [Feature Requests]
I am using `conda`/`mamba` to manage my R dependencies. `conda-forge` allows installing CRAN packages, which are listed under `r-<CRAN package name>`, and likewise, `bioconda` allows installing Bioconductor packages, which are listed under `bioconductor-<Bioconductor package name>`.

Would it be possible to post ArchR to Bioconductor?

(https://github.com/GreenleafLab/ArchR/discussions/1553 is a similar discussion, but suggests uploading ArchR to the `bioconda` repo itself, which is unnecessary).

**Comment by rcorces** [comment]
No. ArchR will not be submitted to Bioconductor any time in the near future.

**Reply by Pandapip1** [reply]
Would you mind explaining why? It would be very helpful!

**Reply by rcorces** [reply]
The potential benefit is very minimal in my mind and Bioconductor prevents packages from having dependencies outside of CRAN/Bioconductor, of which ArchR has 3. ArchR also does not have support for Windows which is required.

**Reply by Pandapip1** [reply]
Fair enough. I would highly recommend creating a bioconda package, then.
